<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="proxy_pages_widget">
     <widget class="QWidget" name="proxy_selection_page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Scale</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="scale_selection">
             <property name="toolTip">
              <string/>
             </property>
             <item>
              <property name="text">
               <string>Regional</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Camp</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Deposit</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Mineral system</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="mineral_system_selection">
             <item>
              <property name="text">
               <string>IOCG</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Li-Pegmatites</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Co-VMS</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTabWidget" name="tabWidget_dock">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="source_tab_dock">
            <attribute name="title">
             <string>Source</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="pathway_tab_dock">
            <attribute name="title">
             <string>Active/structural pathways</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="depositional_tab_dock">
            <attribute name="title">
             <string>Depositional processes</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="mineralisation_tab_dock">
            <attribute name="title">
             <string>Mineralisation, remobilisation, modification</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="proxy_creation_page_1"/>
     <widget class="QWidget" name="proxy_creation_page_2">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>page 2</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="proxy_creation_page_3"/>
     <widget class="QWidget" name="proxy_creation_page_4"/>
     <widget class="QWidget" name="proxy_creation_page_5"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
